codebase/superdarn/src.bin/tk/tool/make_fit.1.19/make_fit.c

```diff
@@ -110,6 +162,26 @@ int main(int argc,char *argv[]) {
   arg=OptionProcess(1,argc,argv,&opt,rst_opterr);
 
   if (arg==-1) {
+     fprintf(stderr,"Error processing options\n");
+      OptionFree(&opt);
+      exit(-1);
+  }
+
+  if (option==1) {
```

Comment:
There is duplication of several of these command line option if statements here with lines 206-225 below.